src/Bookings.js

```diff
@@ -1,18 +1,42 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [loadingData, setLoadingData] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    try {
+      fetch("https://cyf-react.glitch.me/error")
+        .then(res => res.json())
+        .then(data => {
+          setBookings(data);
+          setLoadingData(true);
+        });
+    } catch (error) {
+      setError(true);
+      console.log(error);
+    }
+  }, []);
```

Comment:
This useEffect is looking good, just make sure to check the URL that you are fetching from, if the data type that this URL returns is not the same as what you are expecting, it could cause some issues.